src/countItems.js

```diff
@@ -12,14 +12,23 @@
  *  string in the input array
  */
 function countItems(array) {
-  // This is your job. :)
+  let finalDict = {};
+  for (element of array) {
```

Comment:
Without `let` here, `element` becomes a global variable. Should be `for(let element of array) { ... }`.